src/test/java/org/apache/ibatis/submitted/named_constructor_args/User.java

```diff
@@ -28,7 +28,7 @@ public User(@Param("id") String id) {
     this.id = Integer.valueOf(id);
   }
 
-  public User(Integer userId, @Param("name") String userName) {
+  public User(@Param("userId") Integer userId, @Param("name") String userName) {
```

Comment:
This change seems unnecessary. Please revert or explain why it's necessary.

Reply:
if I did not add the @Param annotation. some of my test can't pass,only add these annotations, maven install can be success.

Reply:
Ah, I see.
These tests require a compiler option `-parameters` and some IDEs require extra configuration.
See [this comment](https://github.com/mybatis/mybatis-3/issues/549#issuecomment-262727988) if you are using Eclipse.

Reply:
nice, I have revert User.java and Mapper.java. now, compile can success.